### ISSUE TITLE
The painting panel is great, but I can't seem to use it as a mask conveniently

### ISSUE THREAD

**Issue by abmmmm** [AlekPet/ComfyUI_Custom_Nodes_AlekPet#3]
After changing the image to the input port, I can't read the generated picture, if I can read the generated image, make a mask on it and output it, I think, it will be great, maybe I didn't find the right way?

**Comment by AlekPet** [comment]
If you change the image to input port, then it implies receiving a list with the names of images in the folder as input (in my case, temp, for LoadImage input). I tried to make a mask, but there is confusion in the paths and I have not figured it out yet. Maybe in the near future I will be able to deal with the mask :)

**You can change the code to save images for PainterNode to input folder:**
# painter_node.py
```
import hashlib
import os
from PIL import Image
import torch
import numpy as np
import folder_paths

class PainterNode(object):
    @classmethod
    def INPUT_TYPES(self):
        # temp_dir = folder_paths.get_temp_directory()

        # if not os.path.isdir(temp_dir):
        #     os.makedirs(temp_dir)

        temp_dir = folder_paths.get_input_directory()
        files = [f for f in os.listdir(temp_dir) if os.path.isfile(os.path.join(temp_dir, f))]
        return {"required":
                {"image": (sorted(files), )},
                }

    RETURN_TYPES = ("IMAGE",)
    FUNCTION = "painter_execute"

    CATEGORY = "AlekPet Nodes/image"

    def painter_execute(self, image):
        # image_path = os.path.join(
        #     folder_paths.get_temp_directory(), image)
        image_path = folder_paths.get_annotated_filepath(image)
        i = Image.open(image_path)
        image = i.convert("RGB")
        image = np.array(image).astype(np.float32) / 255.0
        image = torch.from_numpy(image)[None,]

        return (image,)

    @classmethod
    def IS_CHANGED(self, image):
        # image_path = os.path.join(
        #     folder_paths.get_temp_directory(), image)
        image_path = folder_paths.get_annotated_filepath(image)
        m = hashlib.sha256()
        with open(image_path, 'rb') as f:
            m.update(f.read())
        return m.digest().hex()

```
# painter_node.js
Find next line code in painter_node.js:
```formData.append("type", "temp");```
and remove or comment him 
```// formData.append("type", "temp");```
Then in LoadImage node select Paint_0.png or other Paint_...,png, and change Mask.
![comfyui test](https://i.ibb.co/L0RwKQp/test.jpg)

**Comment by abmmmm** [comment]
Thanks a lot for the confusion and look forward to your plugin updates in the future   :)

**Comment by AlekPet** [comment]
Added the ability to use the Mask Editor, and also changed the save image  folder on INPUT (instead of TEMP).
![Mask](https://i.ibb.co/yqnn1jd/maska-editor.jpg)
